### ISSUE TITLE
[REQUEST] Implement DS Microphone in Lemuroid

### ISSUE THREAD

**Issue by shayisnear** [Swordfish90/Lemuroid#952]
Currently, there is no proper Microphone input for the DS via Lemuroid, so I wanted to ask if it would be possible to add in Mic input.

**Additional context**

**Comment by cxnvorge** [comment]
I agree I can't even finish my phoenix wright save file because of the microphone part

**Comment by Irisol2021** [comment]
I am also waiting for a mic option. I can't continue my game without mic.
